requirements.txt

```diff
@@ -4,14 +4,17 @@ diff-match-patch==20121119
 dj-database-url==0.5.0
 dj-static==0.0.6
 Django==2.0.13
+django-filter==2.0.0
+django-widget-tweaks==1.4.3
 django-debug-toolbar==1.9.1
 django-ical==1.4
 django-recaptcha==1.4.0
 django-registration-redux==2.4
 django-reversion==2.0.13
 django-toolbelt==0.0.1
+django-polymorphic==2.1.2
+djangorestframework==3.9.0
```

Comment:
Is this actually being used anywhere?

Reply:
They are not, djangorestframework is left over from PyAssets, I removed all dependencies for it (that were unused) and removed the dependency. Both are now removed as dependencies